pupil/views/question-layout.ejs

```diff
@@ -40,7 +40,7 @@
     </div>
 </div>
 
-<header role="banner" id="global-header" class="<% if (headerClass) { %><%= headerClass %><% } %>">
+<header id="global-header" class="<% if (headerClass) { %><%= headerClass %><% } %>">
```

Comment:
`role="banner"` is a part of the the `govuk_template` : http://alphagov.github.io/govuk_template/example-mvp.html